docs/app/Examples/elements/Button/States/States.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component} from 'react';
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample';
+
+export default class extends Component {
+  render() {
+    return (
+      <div>
+        <ComponentExample
```

Comment:
Questioning our approach now that I see this all laid out.  Instead of having a wrapper div here, wouldn't it make sense to use the `ExampleSection` in these files that bundle `ComponentExamples`?  So, here we'd have:

```jsx
<ExampleSection title='States'>
  <ComponentExample
    title='Active'
    description='A button can show it is currently the active user selection'
    examplePath='elements/Button/States/Active'
  />
  <ComponentExample
    title='Disabled'
    description='A button can show it is currently unable to be interacted with'
    examplePath='elements/Button/States/Disabled'
  />
  <ComponentExample
    title='Loading'
    description='A button can show a loading indicator'
    examplePath='elements/Button/States/Loading'
  />
</ExampleSection>
```

Then, in `Button/ButtonExamples.js` instead of wrapping every import an `ExampleSection`, we could just have:

```jsx
<div>
  <Types />
  <States />
  <Variations />
</div>
```

Thoughts?